Doc/library/gzip.rst

```diff
@@ -133,15 +135,10 @@ The module defines the following items:
 
    .. attribute:: mtime
 
-      When decompressing, the value of the last modification time field in
-      the most recently read header may be read from this attribute, as an
-      integer.  The initial value before reading any headers is ``None``.
-
-      All :program:`gzip` compressed streams are required to contain this
-      timestamp field.  Some programs, such as :program:`gunzip`\ , make use
-      of the timestamp.  The format is the same as the return value of
-      :func:`time.time` and the :attr:`~os.stat_result.st_mtime` attribute of
-      the object returned by :func:`os.stat`.
+      When decompressing, this attribute is set to last timestamp in the most
+      recently read header.  It is an integer, holding the number of seconds
+      since the Unix epoch (00:00:00 GMT, Jan. 1, 1970).
```

Comment:
```suggestion
      When decompressing, this attribute is set to the last timestamp in the most
      recently read header.  It is an integer, holding the number of seconds
      since the Unix epoch (00:00:00 UTC, January 1, 1970).
```